documentation/asciidoc/computers/software-sources.adoc

```diff
@@ -46,6 +46,14 @@ The following is a curated list of repositories which most people are interested
 https://github.com/raspberrypi/linux::
 This repository contains the source code for the downstream Linux kernel supported and working on all versions of Raspberry Pi from Pi 1 to Pi 5.
 
+https://github.com/raspberrypi/firmware::
+This repository contains the closed source firmware, although the following two links are useful:
```

Comment:
I think this repo supplies the data for rpi-update? It might be worth adding something along those lines to give a better understanding of what the repo is actually for.

Reply:
https://github.com/raspberrypi/rpi-firmware/pull/2